Implementação e estilo

dancarvalho09 opened the pull request:
Implementação do rating bar, favoritar e alteração de estilo na descrição do produto. #31 

@luanah-souzza @janine-oliveira no arquivo `pubspec.yaml` removi o another_carousel_pro que estava comentado em `dev_depencencies` pois quando o criei, ele foi transferido para o `dependencies` e deixei comentado caso acontecesse algum erro no carrosel de imagens na criação da  _home_.

dancarvalho09 committed:
Implementação e estilo

Implementação do rating bar, favoritar e alteração de estilo

luanah-souzza commented on lib/components/product_detail.dart:
```diff
@@ -117,31 +140,57 @@ class ProductDetailState extends State<ProductDetail> {
                   ),
                 ),
-              ),
+                const Padding(padding: EdgeInsets.symmetric(vertical: 10, horizontal: 10)),
```
@dancarvalho09, alterei os valores da linha 79, para 5px, para funcionar em diferentes tamanhos de tela.   
`const Padding(padding: EdgeInsets.symmetric(vertical: **5**, horizontal: **5**)),`
![botão-favoritar](https://github.com/code-fusion-ima/hackadev/assets/80600152/145d9c74-2a4a-4107-8e42-3ca7c39c08ae)

luanah-souzza committed:
style: Update product_detail.dart

Alterei a linha 179 para: 
const Padding(padding: EdgeInsets.symmetric(vertical: 5, horizontal: 5)),
trocando o valor vertical e horizontal de 10 para 5, para melhor visualização em diferentes tamanhos de tela

luanah-souzza reviewed (commented):
Código revisado e alterações feitas!

code-fusion-ima commented:
Código aprovado!
